nemo/collections/nlp/data/language_modeling/megatron/misc_utils.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+
+
+def get_datasets_weights_and_num_samples(data_prefix, train_valid_test_num_samples):
+
```

Comment:
Could you add a short description of the method?

Reply:
This is stuff that already existed and I just moved it from one file to another. But sure, I can add a docstring. Same for all other things in this file.

Reply:
Well appreciated.